typescript-intro/typescript-intro-quiz-notes.md

```diff
@@ -0,0 +1,47 @@
+# typescript-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- When was TypeScript introduced?
+  A: Typescript was created in 2012.
+- Who created TypeScript?
+  A: Microsoft created TypeScript.
+
+- What does the TypeScript `Compiler` do?
+  A: The compiler can convert code from languages like Java and C++ into code a computer can run.
```

Comment:
Right, but a compiler can also just convert one language to another. The TS compiler converts your TS code into JS code so that the browser can run it...